### ISSUE TITLE
Determine if WebGPU is ready to try

### ISSUE THREAD

**Issue by wingman-jr-addon** [wingman-jr-addon/wingman_jr#148]
Browsers and TF.js are both getting further in support. It might be worth trying to see how performance is, if it is possible with current stage of development.

**Comment by wingman-jr-addon** [comment]
I gave this a try over at https://github.com/wingman-jr-addon/wingman_jr/tree/tfjs-webgpu-first-look - Firefox's WebGPU support doesn't seem like it's there yet. I had to enable a bunch of switches in the browser, and then things don't load/work right. I think with Firefox's pace of WebGPU development, it is highly unlikely that WebGPU will become competitive as a backend for Firefox for a long time. Additionally, even on Chrome, I've seen a bit mixed reports over on TF.js about performance - some parts are better, some worse - so it is also not clear that this is a gain overall. For now I'm going to remove this from the long-term roadmap for those reasons.
